CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-01-29
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* Starting with Kubernetes 1.26:
+  * HostProcess Containers will be GA
+  * Some AKS labels will be deprecated. Update your AKS labels to the recommended substitutions. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+* AKS began pod security policy deprecation on 2022-11-01 API. The [pod security policy](https://learn.microsoft.com/azure/aks/use-pod-security-policies) will be removed completely on 2023-06-01 API with AKS 1.25 version or higher. You can migrate pod security policy to [pod security admission controller](https://learn.microsoft.com/azure/aks/use-psa) before the deprecation deadline.
+* Azure Policy will be updated to [GateKeeper 3.11](https://github.com/open-policy-agent/gatekeeper/releases/tag/v3.11.0) on Feb 20th for AKS 1.24 and up.
+* Workload Identity: Application pods using workload identity will need the following label added `azure.workload.identity/use` starting with the 2023-01-29 release. Add the label to your running pods/deployments to avoid pods from failing at restart. See more [here](https://learn.microsoft.com/azure/aks/workload-identity-overview#service-account-labels).
+
+### Release notes
+
+* Features
+  * New k8s patch versions for 1.23: Added 1.23.15, removed 1.23.8
+* Preview Feature
+  * Azure CNI Overlay now available in uksouth, australiaeast 
+* Behavior Change
+  * AKS now allows updating httpProxyConfig.noProxy after cluster deployment. This updates the webhook which automatically injects environment variables into pods. It does not yet update noProxy configuration on running nodes. New nodepools or VMSS model upgrades will update noProxy on the nodes. In the future, rolling nodes may become automatic.
```

Comment:
doc update is forthcoming.

Reply:
Good catch.

That description is both too low level and not detailed enough. Also this is a feature, not a behavior change.

` AKS now allows updating the noProxy value of HTTP Proxy configuration after cluster deployment. This will automatically inject new environment variables into pods with the new noProxy values.  Pods must be rotated for the apps to pick it up.
For components under kubernetes, like containerd and the node itself this won't take effect until a node image upgrade is performed.`

Reply:
Requesting changes on this

Reply:
well technically, we are changing the behavior of an http proxy to allow update, which removes a limitation